proto/etcd.proto

```diff
@@ -2,6 +2,12 @@ syntax = "proto3";
 
 package etcdserverpb;
 
+service KV {
+  rpc Put(PutRequest) returns (PutResponse) {}
+  rpc Range(RangeRequest) returns (RangeResponse) {}
+  rpc DeleteRange(DeleteRangeRequest) returns (DeleteRangeResponse) {}
+}
+
```

Comment:
Why is this repeated here?

Reply:
k6's grpc support relies on reading rpc definitions from the proto file. I guess we could just point it to the etcd file but that might confuse it if we want to test out any extensions we've added (e.g. response headers having committed revisions) so I added the service definition here.